src/transformer/utils/get_activity/course.php

```diff
@@ -22,30 +22,25 @@
 function course(array $config, \stdClass $course) {
     $coursename = $course->fullname ? $course->fullname : 'A Moodle course';
     $courselang = utils\get_course_lang($course);
-    $sendshortid = utils\is_enabled_config($config, 'send_short_course_id');
 
-    if ($sendshortid) {
-        return [
-            'id' => $config['app_url'].'/course/view.php?id='.$course->id,
-            'definition' => [
-                'type' => 'http://id.tincanapi.com/activitytype/lms/course',
-                'name' => [
-                    $courselang => $coursename,
-                ],
-                'extensions' => [
-                    'https://w3id.org/learning-analytics/learning-management-system/short-id' => $course->shortname
-                ]
-            ],
-        ];
+    $object = [
+                  'id' => $config['app_url'].'/course/view.php?id='.$course->id,
+                  'definition' => [
+                      'type' => 'http://id.tincanapi.com/activitytype/lms/course',
+                      'name' => [
+                          $courselang => $coursename,
+                      ],
+                  ],
+              ];
+
+    if (utils\is_enabled_config($config, 'send_short_course_id')) {
```

Comment:
Usually prefer to avoid mutations because it becomes quite hard to read as the size of the code grows, but that is quite painful in PHP (usually work in JS where this is easier to avoid). Happy with this though, just thought I'd explain why it wasn't like that before, just a personal habit.